CRM/Upgrade/Incremental/php/FiveFiftyNine.php

```diff
@@ -30,6 +30,17 @@ class CRM_Upgrade_Incremental_php_FiveFiftyNine extends CRM_Upgrade_Incremental_
   public function upgrade_5_59_alpha1($rev): void {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
     $this->addTask('Drop column civicrm_custom_field.mask', 'dropColumn', 'civicrm_custom_field', 'mask');
+    $this->addTask('Add scheduled_reminder_smarty setting', 'addScheduledReminderSmartySetting');
+  }
+
+  public function setPostUpgradeMessage(&$postUpgradeMessage, $rev): void {
+    if ($rev === '5.59.alpha1') {
+      $postUpgradeMessage .= 'You can now choose whether to use Smarty in Scheduled Reminders via the scheduled_reminder_smarty setting. We have enabled the setting during this upgrade to preserve the existing behaviour but you may wish to disable it now. Note: the setting is disabled by default in new installations. More information here: https://lab.civicrm.org/dev/core/-/issues/4100.';
```

Comment:
Needs ts(). Also I know it's picky but since there isn't a way to translate _to_ en_US, the original text should have "behavior".

Reply:
Thanks - done.